mips/input4MIPs/index.md

```diff
@@ -4,16 +4,28 @@ title: PCMDI - input4MIPs
 ---
 
 # input4MIPs
-input4MIPS (input datasets for Model Intercomparison Projects) is an activity
+input4MIPs (input datasets for Model Intercomparison Projects) is an activity
 to make available via ESGF the boundary condition and forcing datasets needed
-for CMIP6. Various datasets are needed for the pre-industrial control (piControl),
-AMIP, and historical simulations, and additional datasets are needed for
-many of the [CMIP6-endorsed model intercomparison projects (MIPs) experiments][CMIP6EndorsedMIPs].
+for Model Intercomparison Projects (MIPs). Various datasets are needed for the
+CMIP Diagnostic, Evaluation and Characterization of Klima (DECK) experiments
+including the pre-industrial control (piControl), amip and historical experiments.
+
+Work is underway on next-generation forcing datasets to meet the experimental
+needs of the upcoming CMIP7 project phase. For more information, see
+[input4mips-cvs.readthedocs.io](input4mips-cvs.readthedocs.io/). Prototype data
+are currently available, [published to the input4MIPs CMIP6Plus MIP era on ESGF][CMIP6Plus].
+
+Earlier versions of forcing data were prepared for the preceding phase, CMIP6,
+and were also [published to the input4MIPs CMIP6 MIP era on ESGF][CMIP6], which
+include data for the CMIP6 DECK experiments, and additional datasets were
+prepared for many of the [CMIP6-endorsed model intercomparison projects (MIPs) experiments][CMIP6EndorsedMIPs].
 
 Earlier versions of many of these datasets were used in the [5th Coupled Model Intercomparison Project (CMIP5)][CMIP5].
 
 [More Information][input4MIPs]
 
 [CMIP6EndorsedMIPs]: http://www.wcrp-climate.org/modelling-wgcm-mip-catalogue/modelling-wgcm-cmip6-endorsed-mips
 [CMIP5]: /mips/cmip5
-[input4MIPs]: https://esgf-node.llnl.gov/projects/input4mips
+[CMIP6]: https://aims2.llnl.gov/search?project=input4MIPs&activeFacets={%22mip_era%22:%22CMIP6%22}
```

Comment:
Are these characters "{:}" causing issues with the build?

Reply:
The `{%` in the URLs are causing problems with Jekyll.  They need to be placed within raw tags.
```
{% raw %} <url> {% endraw %}
```
I'll have a pull request up shortly to fix the URLs.